cluster/gce/config-default.sh

```diff
@@ -16,23 +16,23 @@
 
 # TODO(jbeda): Provide a way to override project
 # gcloud multiplexing for shared GCE/GKE tests.
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
 source "${KUBE_ROOT}/cluster/gce/config-common.sh"
 
 # Specifying KUBE_GCE_API_ENDPOINT will override the default GCE Compute API endpoint (https://www.googleapis.com/compute/v1/).
 # This endpoint has to be pointing to v1 api. For example, https://www.googleapis.com/compute/staging_v1/
-GCE_API_ENDPOINT=${KUBE_GCE_API_ENDPOINT:-}
-GCLOUD=gcloud
+export GCE_API_ENDPOINT=${KUBE_GCE_API_ENDPOINT:-}
```

Comment:
shellcheck said SC2034, so I added `export`. The other lines are the same.
```
GCE_API_ENDPOINT=${KUBE_GCE_API_ENDPOINT:-}
^--------------^ SC2034: GCE_API_ENDPOINT appears unused. Verify use (or export if used externally)
```

Reply:
this one is in fact read by other processes (see the comment above)

Reply:
we should prefer verifying that these are used and remove them if not to just adding export to silence shellcheck

cs.k8s.io can help

Reply:
I've checked all variables.
Only `SERVICE_CLUSTER_IP_SUBNETWORK` is not used anywhere.
https://cs.k8s.io/?q=SERVICE_CLUSTER_IP_SUBNETWORK&i=nope&files=&repos=

So, I will remove it.

Reply:
I updated this commit.

Reply:
cc @cheftako